packages/lib/src/classes/ApplicationCommand.ts

```diff
@@ -0,0 +1,33 @@
+import {
+	ApplicationCommandOptionData,
+	ApplicationCommandType,
+	AutocompleteFocusedOption,
+	AutocompleteInteraction,
+	ChatInputCommandInteraction,
+	MessageContextMenuCommandInteraction,
+	UserContextMenuCommandInteraction
+} from "discord.js"
+import { BetterClient, ApplicationCommandOptions } from "../index.js"
+import BaseComponent from "./_BaseComponent.js"
+
+export default class ApplicationCommand extends BaseComponent {
+	public readonly description: string
+	public readonly type: ApplicationCommandType = ApplicationCommandType.ChatInput
+	public readonly options: ApplicationCommandOptionData[] = []
```

Comment:
I would really love if this used builders and not objects. So much cleaner.

Reply:
I've never been a fan of builders, and this lib will support guilded.js as well so we're gonna stick to this to make that easier.